### ISSUE TITLE
User Sync Tools Error

### ISSUE THREAD

**Issue by tinnesb** [adobe-apiplatform/user-sync.py#439]
Hi, Posted this to the official Adobe support team, and they redirected to here (great support Adobe!)

When running the   user-sync.pex file from windows, we receive the following error:   FileNotFoundError: [Errno 2]   No such file or directory: 'C:\\Users\\Username\\.pex\\install\\pycryptodome-3.4.7-cp36-cp36m-win_amd64.whl.ef643e79769d061fe   8a935f3dd3ed41ad6d5f2f9.ddfbd24020c34d4fb919c6b55246b6d0\\pycryptodome-3.4.7-cp3   6-cp36m-win_amd64.whl\\Crypto/SelfTest/Cipher/__pycache__/test_Blowfish.cpython-   36.pyc'   

We have set an environment   variable using: setx -m PEX_ROOT C:\pex but still receive the error   

Thanks,
Tim

**Comment by adorton-adobe** [comment]
I'm not familiar with `setx` but it doesn't appear to add the variable in the current console session.  Start a new console session and try again.  Also check the environment variables UI to ensure it was created successfully.  You can also check in powershell - 

```
> Get-ChildItem Env:
```

If it didn't create the variable, it may be due to the fact that `setx` expects `/m` and not `-m`.

https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/setx#BKMK_examples

**Comment by adorton-adobe** [comment]
@tinnesb did that resolve your issue?

**Comment by tinnesb** [comment]
@adorton-adobe,

No, the variables where all correct, but I think I have it working now using the MSI installer. Happy to close this off.

Thanks,
